airbyte-cdk/java/airbyte-cdk/typing-deduping/src/testFixtures/kotlin/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.kt

```diff
@@ -402,26 +406,76 @@ abstract class BaseSqlGeneratorIntegrationTest<DestinationState : MinimumDestina
      */
     @Test
     @Throws(Exception::class)
-    fun allTypes() {
-        // Add case-sensitive columnName to test json path querying
-        incrementalDedupStream.columns!!.put(
-            generator.buildColumnId("IamACaseSensitiveColumnName"),
-            AirbyteProtocolType.STRING
+    fun allTypes_safeCast() {
+        assumeTrue(
+            supportsSafeCast,
+            "Skipping test because this connector does not support safe casting"
         )
         createRawTable(streamId)
         createFinalTable(incrementalDedupStream, "")
         insertRawTableRecords(
             streamId,
-            BaseTypingDedupingTest.Companion.readRecords("sqlgenerator/alltypes_inputrecords.jsonl")
+            Streams.concat(
```

Comment:
You should be able to use `readRecords + readRecords`, with the calls to `stream()` and `toList()`

Reply:
nice! (it's going to take me forever to unlearn these javaisms)